go-common.mk

```diff
@@ -75,7 +75,7 @@ build:: pre-build standard-build post-build
 
 pre-build::
 
-standard-build:: $(_GO_ROOT_BUILD_TARGET) $(_GO_BUILD_TARGETS)
+standard-build:: $(_GO_ROOT_BUILD_TARGET) $(_GO_BUILD_TARGETS) go.mod
```

Comment:
If I understand the aim here, I think this should also be added to the lines further down that look like

```make
$(_GO_ROOT_BUILD_TARGET): $(GOSRC)

...

$(_GO_BUILD_TARGETS): $(GOSRC)
```

i.e., they should become

```make
$(_GO_ROOT_BUILD_TARGET): $(GOSRC) go.mod

...

$(_GO_BUILD_TARGETS): $(GOSRC) go.mod
```

Otherwise this is only taken into account for library projects.